assets/prototype/application/services/apollo/mutations/types.ts

```diff
@@ -48,7 +48,11 @@ export interface EntityMutation {
 	mutate: (options: CustomMutationOptions) => MutationResult;
 }
 
-export interface Mutators {
+interface MutatorsMap {
+	[key: string]: Mutator;
+}
+
+export interface Mutators extends MutatorsMap {
```

Comment:
Fix for the 
![14_jan_7](https://user-images.githubusercontent.com/1477453/72360192-483d7d00-36f8-11ea-9df9-32722555867f.png)